reports/California.md

```diff
@@ -458,6 +458,31 @@ id: ca-losangeles-22
 * https://twitter.com/josie_huang/status/1274863117004988416
 
 
+### Los Angeles law enforcement beat protesters with batons while they have their hands up | May 30th
```

Comment:
hey can you put these in the right chronological position in LA?

Sorry i probably wasn't clear initially, it's alphabetically by City, then chronological within city